wagtail_calendar/wagtail_hooks.py

```diff
@@ -1,21 +1,32 @@
 from django.conf.urls import url, include
+from django.core.exceptions import ImproperlyConfigured
 from django.urls import reverse
 from django.utils.dateparse import parse_datetime
 from django.utils.translation import ugettext_lazy as _
-from wagtail.wagtailadmin.menu import MenuItem
-from wagtail.wagtailcore import hooks
-from wagtail.wagtailcore.models import Page, PageRevision, UserPagePermissionsProxy
+
+try:
+    from wagtail.admin.menu import MenuItem
+    from wagtail.core import hooks
+    from wagtail.core.models import Page, PageRevision, UserPagePermissionsProxy
+except:
+    from wagtail.wagtailadmin.menu import MenuItem
+    from wagtail.wagtailcore import hooks
+    from wagtail.wagtailcore.models import Page, PageRevision, UserPagePermissionsProxy
 
 from wagtail_calendar import urls
 from wagtail_calendar.utils import get_page_event_data
 
 
 @hooks.register('register_admin_urls')
 def register_admin_urls():
-    return [
-        url(r'^calendar/', include(urls, namespace='wagtail_calendar')),
-    ]
-
+    try:
+        return [
+            url(r'^calendar/', include(urls, namespace='wagtail_calendar')),
+        ]
+    except ImproperlyConfigured:
+        return [
+            url(r'^calendar/', include((urls,'wagtail_calendar'), namespace='wagtail_calendar')),
```

Comment:
Why did you have to do that? `urls` should be a list of urls in this case and not the module itself.

Reply:
I see what the issue is actually now.